status_codes/error.go

```diff
@@ -52,7 +52,9 @@ func WrapError(err error, code int, msg string) *ProtocolError {
 
 */
 var (
-	ErrSerialization = ProtocolError{InternalErrorSerialization, "error in serialization"}
+	ErrSerialization      = ProtocolError{InternalErrorSerialization, "error in serialization"}
+	ErrFrozenValidator    = ProtocolError{InternalErrorSerialization, "error frozen validator"}
```

Comment:
Need to define new error codes for 'ErrFrozenValidator' and 'ErrNonFrozenValidator'